tests/woocommerce/cypress/integration/shop/chile.js

```diff
@@ -17,7 +17,7 @@ const mock = (data) => (R.merge(
     city: Faker.address.city(),
     state: Faker.address.state(),
     zipcode: Faker.address.zipCode(),
-    document: Faker.random.alpha({count: 9}),
+    document: '123456789',
```

Comment:
Why you are using a fix document and not a Faker?

Reply:
@Ruanito intermittent tests, because Faker sometimes has the alpha method and sometimes don't. We use fix document in Volcanes 😄